src/components/list-with-actions/_list-with-actions.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../../../govuk-frontend/govuk/settings/all";
+@import "../../../../govuk-frontend/govuk/tools/all";
+@import "../../../../govuk-frontend/govuk/helpers/all";
+
+.hmrc-list-with-actions {
+  @include govuk-font($size: 19);
+}
+
+// Kept for backward compatibility with add-to-a-list
+.hmrc-list-with-actions--short {
+  @include govuk-media-query($from: desktop) {
+    .hmrc-list-with-actions__name {
+      width: 30%;
+    }
+  }
+}
+
+// Kept for backward compatibility with add-to-a-list
+.hmrc-list-with-actions--long {
+  @include govuk-media-query($from: desktop) {
+    .hmrc-list-with-actions__name {
+      width: 50%;
+    }
+  }
+}
+
+.hmrc-list-with-actions__item {
+  display: flex;
```

Comment:
Using flexbox instead of a table layout due to the possibility of different numbers of actions.